sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -285,6 +292,38 @@ export interface KeyReleasePolicy {
     data?: Uint8Array;
 }
 
+// @public
+export interface KeyRotationLifetimeActions {
+    action?: "Rotate" | "Notify";
+    trigger?: KeyRotationLifetimeActionsTrigger;
+}
+
+// @public
+export interface KeyRotationLifetimeActionsTrigger {
+    timeAfterCreate?: string;
+    timeBeforeExpiry?: string;
+}
+
+// @public
+export interface KeyRotationPolicy {
+    attributes?: KeyRotationPolicyAttributes;
+    readonly id?: string;
+    lifetimeActions?: KeyRotationLifetimeActions[];
+}
+
+// @public
+export interface KeyRotationPolicyAttributes {
```

Comment:
We don't expose an "Attributes" class already, but do expose "Properties". I think this should be `KeyRotationPolicyProperties` thus.

On top of that, the members should match what we use for `KeyProperties` already e.g., `createdOn`, `updatedOn`, and maybe `expiresOn` though, given it's a string, I assume it's a ISO8601 duration which we don't fully support across SDKs. Thus, "On" may not work here. We should discuss. Given the structure of other property names, maybe "expiresIn"?

Also, most - if not all of our languages - have some sort of time span I wonder if we should support instead. That said, though, it was @JeffreyRichter who had concerns about Go not supporting ISO8601 durations well. Maybe a `string` here is best, but not very idiomatic, IMO.